lsp/glint.lua

```diff
@@ -0,0 +1,63 @@
+---@brief
+---
+--- https://github.com/typed-ember/glint
+--- https://typed-ember.gitbook.io/glint/
+--- `glint-language-server` is installed when adding `@glint/core` to your project's devDependencies:
+---
+--- ```sh
+--- npm install @glint/core --save-dev
+--- yarn add -D @glint/core
+---
+--- This configuration uses the local installation of `glint-language-server`
+--- (found in the `node_modules` directory of your project).
+---
+--- To use a global installation of `glint-language-server`,
+--- set the `init_options.glint.useGlobal` to `true`.
+---
+--- vim.lsp.config('glint', {
+---   init_options = {
+---     glint = {
+---       useGlobal = true,
+---     },
+---   },
+--- })
+
+return {
+  cmd = { 'glint-language-server' },
+  init_options = {
+    glint = {
+      useGlobal = false,
+    },
+  },
+  before_init = function(_, config)
+    if config.init_options.glint.useGlobal then
+      return
+    end
+
+    local root_dir = config.root_dir
+    if not root_dir then
+      error('No root directory found for glint')
+    end
+    config.cmd = {
+      'exec',
+      root_dir .. '/node_modules/.bin/glint-language-server',
```

Comment:
I don't think this will do what is expected. `before_init` happens after the server was starting.

If `cmd` needs to be dynamic it must be defined as a function.